env.example

```diff
@@ -3,3 +3,7 @@ NODE_ENV=development
 
 # PORT is the port used by the web server
 PORT=8080
+
+# INACTIVETIME is the period (millisecond) of inactivity
+# before a blog will be redlisted
+INACTIVETIME=31556952000
```

Comment:
`BLOG_INTERACTIVE_TIME`

Reply:
I'll address this, but before I do, did you mean `BLOG_INACTIVE_TIME`?

Reply:
Yeah, sorry, `BLOG_INACTIVE_TIME`, good catch.